app/views/questions/new.html.erb

```diff
@@ -8,7 +8,7 @@
             <%= form_for(@question) do |f| %>
               <%= f.text_field :text, placeholder: 'Question',  class: 'form-control', autofocus: true %>
               <%= f.label :qset_id, 'Qset' %>
-              <%= f.select :qset_id, @qsets.map {|qs| [qs.name, qs.id]} %>
+              <%= f.collection_select(:qset_id, Qset.find(current_user.org).descendants, :id, :name) %>
```

Comment:
We've noted before that some users don't yet have an `org` in prod. So when we upgrade prod we'll have to either make sure there's a default for logic like this, or we'll have to ensure every existing user gets put in an org (in which case the qsets_controller.rb code in the `if current_user.org.nil?` block really isn't necessary anymore). It's probably better to go the latter route, because that way we can assume everyone has an org from now on. So after 'thinking out loud' I think it's ok to leave this as-is.